<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualMachine</class>
 <widget class="QMainWindow" name="VirtualMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VirtualMachine</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>yyx.ico</normaloff>yyx.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>631</width>
      <height>111</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>未选择文件</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRun">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>qemu-system-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>270</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>启动方式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>310</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>内存：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMenory">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>启动虚拟机后关闭窗口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ISO：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonISO">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>未选择镜像</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMachine">
    <property name="title">
     <string>虚拟机</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionQt"/>
    <addaction name="separator"/>
    <addaction name="actionMe"/>
    <addaction name="actionGet"/>
   </widget>
   <addaction name="menuMachine"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>新虚拟机</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionMe">
   <property name="text">
    <string>关于本软件</string>
   </property>
  </action>
  <action name="actionGet">
   <property name="text">
    <string>获取源代码</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
